linkcheck/configuration/__init__.py

```diff
@@ -365,9 +366,12 @@ def sanitize_ssl(self):
 
 
 def get_plugin_folders():
-    """Get linkchecker plugin folders. Default is ~/.linkchecker/plugins/."""
+    """Get linkchecker plugin folders. Default is
+    $XDG_DATA_HOME/linkchecker/plugins/."""
```

Comment:
Maybe mention in the docstring that ~/.linkchecker/plugins/ is also supported for backwards compatibility, and what happens if both exist?